dotnet/src/SemanticKernel.Abstractions/AI/AIException.cs

```diff
@@ -49,7 +49,7 @@ public AIException(ErrorCodes errorCode, string? message, Exception? innerExcept
     /// <param name="message">A string that describes the error.</param>
     /// <param name="detail">A string that provides additional details about the error.</param>
     public AIException(ErrorCodes errorCode, string? message, string? detail)
-        : this(errorCode, message, detail: null, innerException: null)
```

Comment:
Sorry I missed this.

Seems like we should augment the tests for all of these.

Reply:
Since I last touched them, I'll add tests after this merges.

Reply:
We all missed it, no sweat :) Thanks for offering to add more though, a good idea.